src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -49,7 +49,7 @@ export const ProjectFeatureToggles = ({
             headerContent={
                 <HeaderTitle
                     className={styles.title}
-                    title={`Feature toggles (${filteredFeatures.length})`}
+                    title={`Project features (${filteredFeatures.length})`}
```

Comment:
For consistency, should this be "project feature toggles"? I think we used "feature toggles" further up 🤔